packages/scene-composer/src/augmentations/components/three-fiber/anchor/AnchorWidget.tsx

```diff
@@ -104,11 +106,13 @@ export function AsyncLoadedAnchorWidget({
 
   // Evaluate visual state based on data binding
   const visualState = useMemo(() => {
```

Comment:
Is this variable only consumed in testing? It seems like it's also only the icon state now as it has no color information in the actual return value.

Reply:
If I understand correctly your comment, you are asking about custom color info for the icon.  If it is Info then it will be iottwinmaker-common-icon:Info. For custom color it will be iottwinmaker-common-icon:Custom-#ffffff. Color information will be part of the targetIcon value.

Reply:
Ah, okay. Wasn't clear from just scanning variable names. That was one part.

Reply:
Other question is do we use the const visualState anywhere?

Reply:
Yes we pass this visualState part of anchor widget prop.

Reply:
Okay, missed expanding one code section, found it.